docs/api/api-reference.md

```diff
@@ -318,6 +318,7 @@ _Appears in:_
 | `transformationRefs` _[TransformationRef](#transformationref) array_ | TransformationRefs contain references to template configuration from SecretTransformation |
 | `includes` _string array_ | Includes contains regex patterns of keys that should be included in the K8s Secret Data. FieldFilter can be used to filter the secret data that is stored in the K8s Secret Destination. Filters will not be applied to templated fields, those will always be included in the Destination K8s Secret. Exclusion filters are always applied first. |
 | `excludes` _string array_ | Excludes contains regex pattern for keys that should be excluded from the K8s Secret Data. FieldFilter can be used to filter the secret data that is stored in the K8s Secret Destination. Filters will not be applied to templated fields, those will always be included in the Destination K8s Secret. Exclusion filters are always applied first. |
+| `excludeRaw` _boolean_ | ExcludeRaw data from the destination Secret. Exclusion policy can be set globally by including 'exclude-raw` in the '--global-rendering-options' command line flag. The global policy always takes precedence over this configuration. |
```

Comment:
```suggestion
| `excludeRaw` _boolean_ | ExcludeRaw data from the destination Secret. Exclusion policy can be set globally by including 'exclude-raw` in the '--global-rendering-options' command line flag. If set, the command line flag always takes precedence over this configuration. |
```

Reply:
This bit is auto generated from the structure member's doc strings. I can updated it there.

Reply:
Updated in 8f2fa91c52b3e7b1b60a4f5b87c84868d35e166c